wp/plugin/wp.ml

```diff
@@ -218,7 +218,7 @@ let func_name_map = Cmd.parameter Typ.(list ~sep:';' (pair ~sep:',' string strin
            "<reg1_orig>,<reg1_mod>;<reg2_orig>,<reg2_mod>"). By default, WP
            assumes subroutines have the same names between the two binaries.|}
 
-let user_func_spec = Cmd.parameter Typ.(some (t3 string string string)) "user-func-spec"
+let user_func_spec = Cmd.parameter Typ.(list ~sep:';' (t3 string string string)) "user-func-spec"
```

Comment:
Should it still be `user-func-spec` or should it be changed to `user-func-specs` (with the `s` at the end, or does it not matter)? Does anyone have thoughts?

Reply:
I have thoughts: it does matter (but not a lot!), and it should be `user_func_specs`.

Reply:
or `user-func-specs` rather, though I'd probably rename the OCaml as well.